mvt/feature.go

```diff
@@ -519,11 +519,16 @@ func createDebugFile(min, max maths.Pt, geo tegola.Geometry, err error) {
 		return
 	}
 	filename := fmt.Sprintf("/tmp/testcase_%v_%p.json", fln, geo)
-	bgeo, err := basic.CloneGeometry(geo)
+	geo, err = basic.CloneGeometry(geo)
 	if err != nil {
 		log.Errorf("failed to clone geo for test case. %v", err)
 		return
 	}
+	bgeo, err := convert.ToTegola(geo)
```

Comment:
This goes away in https://github.com/go-spatial/tegola/pull/624 so we will have a conflict to deal with. should be minor.